src/objects/form-group/form-group.twig

```diff
@@ -1,6 +1,8 @@
 <label class="o-form-group{% if class %} {{ class }}{% endif %}">
-  {% block label %}
-    <div>{{ label | default('Label') }}</div>
-  {% endblock %}
+  <div class="o-form-group__label">
+    {% block label %}
+      {{ label | default('Label') }}
+    {% endblock %}
+  </div>
```

Comment:
I considered whether to make this a breaking change or not, but decided against it:
- I couldn't find any existing places where we were setting `label` via the block.
- Even if we are, the worst that can happen is for the label content to have an extra wrapping `div`, which won't matter since the container is now a flex box.